salvo/src/lib/test_run_benchmark.py

```diff
@@ -151,7 +167,7 @@ def test_benchmark_failure_if_no_benchmark_selected():
     _ = run_benchmark.BenchmarkRunner(job_control)
 
   assert str(not_implemented.value) == \
-      "No [Unspecified Benchmark] defined"
+      "No [Unspecified Benchmark] defined yet"
```

Comment:
Can you double check that we aren't overwriting changes from previous PR's?  This is another instance where we are reverting changes already upstreamed.

Reply:
There were some merge conflicts between our copy of the repo and what has been upstreamed. For some reason, I thought it was new stuff that we had wanted to upstream, but I was wrong about that.

Should be all fixed now though :)